exonum/src/blockchain/config.rs

```diff
@@ -144,6 +166,39 @@ impl ConsensusConfig {
     /// Time that will be added to round timeout for each next round in terms of percent of first_round_timeout.
     pub const TIMEOUT_LINEAR_INCREASE_PERCENT: u64 = 10; // 10%
 
+    /// Creates a new `ConsensusConfig` object.
+    #[allow(clippy::too_many_arguments)]
+    pub fn new(
```

Comment:
Could it make sense to introduce a builder here? I don't think having a method with 10 args is user-friendly 🙃